### PR TITLE
xsdk-examples: add v0.3.0

### PR DESCRIPTION
**Pull request by balos1**
CC @ulrikeyang

### CODE REVIEW

**Review by balay — DISMISSED** [review]

**Review by luszczek — DISMISSED** [review]

**Review by gardner48 — DISMISSED** [review]
LGTM

File: var/spack/repos/builtin/packages/sundials/package.py [hunk 1]
```diff
@@ -246,6 +246,7 @@ class Sundials(CMakePackage, CudaPackage, ROCmPackage):
     patch('0001-add-missing-README-to-examples-cvode-hip.patch', when='@5.6.0:5.7.0')
     # remove sundials_nvecopenmp target from ARKODE SuperLU_DIST example
     patch('remove-links-to-OpenMP-vector.patch', when='@5.5.0:5.7.0')
+    patch('sundials-v5.8.0.patch', when='@5.8.0')
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/sundials/package.py:250]: Can you add a brief comment on what the patch does (and provenance if you have it) above the directive?

**Review by alalazo — CHANGES_REQUESTED** [review]
Just a minor request, otherwise LGTM

File: var/spack/repos/builtin/packages/sundials/package.py (hunk 1 above)
Comment (balos1) [reply var/spack/repos/builtin/packages/sundials/package.py:250]: @alalazo done

**Review by alalazo — APPROVED** [review]

**Pull request merged**